### PR TITLE
Update: [Blog] Meet The Maintainer : Jash Jash #1789

### LINKED ISSUE #1789: [Blog] Meet The Maintainer : Jash Patel
**Current Behavior**

Maintainer Update : Meet The Maintainer Jash Patel is a new blog post .
It's not on the layer5.io website 

**Desired Situation**

To make a new blog post & add it to [layer5.io](https://layer5.io)
Refer to this doc for more info 
https://docs.google.com/document/d/1o5MasuMXM8unApoTaqrscZy34BQe0_-mdAvmP-6qqjc/edit

Here is a link to the cover photo
https://layer5.io/static/551ae052af26d99e23ae0ab5a053815e/I%20Jash%20Patel.png

Here is blog post that you can refer to solve this issue
https://layer5.io/blog/community/meet-the-maintainer-michael-gfeller

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by blossom-babs**
Meet The Maintainer Jash Patel is a new blog post, which has been added to the 
meet-the-maintainer
layer5.io website

This PR fixes 
#1789

- [ *] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 91d1f1ed19710a38ecad0ad5f147c83183df52cd at: https://6081476ab5a2b00095ae52d5--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Perfect! @blossom-babs

**Comment by chinmaym07** [comment]
Screenshot_2021-04-22 Meet the Maintainer Jash Patel
Some space required here

**Comment by Nikhil-Ladha** [comment]
> Screenshot_2021-04-22 Meet the Maintainer Jash Patel
> Some space required here

Where do you consider adding space. It looks fine to me 🤔

**Comment by chinmaym07** [comment]
Between 'service mesh performance' & 'and'

**Comment by Nikhil-Ladha** [comment]
> Between 'service mesh performance' & 'and'

Ohh.....I have to order a spectacle for myself 😂

**Comment by Nikhil-Ladha** [comment]
@blossom-babs please rectify that.

**Comment by leecalcote** [comment]
MeshMate @Jashpatel1 by MeshMate @Ruth-ikegah! This may be too much. 😄

**Review by Ruth-ikegah — CHANGES_REQUESTED** [review]
Hi @blossom-babs Nice work on this. I have some requested changes:

- Link Layer5 website at the top
- You forgot the "call to action": Be like Jash (chack the google doc)

**Comment by Nikhil-Ladha** [comment]
> Hi @blossom-babs Nice work on this. I have some requested changes:
> 
> * Link Layer5 website at the top
Is that really needed, considering we have it on layer site only.

> * You forgot the "call to action": Be like Jash (chack the google doc)

**Comment by Ruth-ikegah** [comment]
> > Hi @blossom-babs Nice work on this. I have some requested changes:
> > 
> > * Link Layer5 website at the top
> 
> Is that really needed, considering we have it on layer site only.
> 
> > * You forgot the "call to action": Be like Jash (chack the google doc)

oh yeah, skipped that, thanks for the point out!

**Comment by Nikhil-Ladha** [comment]
DCo test is failing @blossom-babs , please check out this link to rectify it https://github.com/layer5io/layer5/pull/1806/checks?check_run_id=2411081050.
Also, can't that space be added without using CSS, simply adding a space in the HTML or the MDX itself, isn't fixing that?

File: src/collections/blog/Blog.style.js [hunk 1]
```diff
@@ -1,207 +1,206 @@
 import styled from "styled-components";
```
Comment (leecalcote) [inline src/collections/blog/Blog.style.js:1]: @blossom-babs were any changes done to this file? If so, could you highlight the line(s) for review? If not, perhaps, you might remove this file from the PR (revert the change on this file). 

Thank you for getting this posted, Blossom. 💯

Comment (Nikhil-Ladha) [reply src/collections/blog/Blog.style.js:1]: The change here that she did was adding `line-spacing` to the `p` elements, which shouldn't be done, here's the [commit](https://github.com/layer5io/layer5/pull/1806/commits/a0accad71ce04b2602f509971c6dd5b91426fafe). And instead, the MDX should be updated.

Comment (blossom-babs) [reply src/collections/blog/Blog.style.js:1]: @Nikhil-Ladha I added line spacing to the hero,  do you suggest I remove the style?

**Comment by blossom-babs** [comment]
> > > Hi @blossom-babs Nice work on this. I have some requested changes:
> > > 
> > > * Link Layer5 website at the top
> > 
> > 
> > Is that really needed, considering we have it on layer site only.
> > > * You forgot the "call to action": Be like Jash (chack the google doc)
> 
> oh yeah, skipped that, thanks for the point out!



> Hi @blossom-babs Nice work on this. I have some requested changes:
> 
> * Link Layer5 website at the top
> * You forgot the "call to action": Be like Jash (chack the google doc)

Right, seen that! nice catch!

File: src/collections/blog/Blog.style.js (hunk 1 above)
Comment (Tanuj22) [reply src/collections/blog/Blog.style.js:1]: @blossom-babs you just need to add a space between 'service mesh performance' & 'and' and not the entire hero.

File: src/collections/blog/2021-04-16-meet-the-maintainer-jash-patel/index.mdx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  <span>Jash:</span>
+  <p>
+    It was a pretty big task. Many people have worked on it(At the time of
+    launch we had 140 contributors). It started with Tanuj and me, and then got
+    many hands to help us and strive towards its completion. I can’t write all
+    those names, or else it would take an entire page to note them down. And
+    this is what communities are meant for; open source is intended for. It was
+    a combined effort, and we worked on almost everything. The team worked on
+    Gatsby, GraphQL, SEO, Analytics, and anything related to the website. The
+    two mentors Lee and Nikhil, were quite helpful for the team to bring it
```
Comment (Jashpatel1) [inline src/collections/blog/2021-04-16-meet-the-maintainer-jash-patel/index.mdx:125]: @blossom-babs Great work overall. Thank you for this.
Can you add profile links to Lee and Nikhil as well?

File: src/collections/blog/Blog.style.js [hunk 3]
```diff
@@ -1,207 +1,206 @@
...
 export const BlogWrapper = styled.div`
-    color:#000;
```
Comment (Tanuj22) [inline src/collections/blog/Blog.style.js:3]: Were any changes done to this file? If no, please revert this.

**Comment by Tanuj22** [comment]
@blossom-babs any updates here. If you are having trouble resolving the merge conflict, you can open up a new PR

File: src/collections/blog/2021-04-16-meet-the-maintainer-jash-patel/index.mdx (hunk 2 above)
Comment (blossom-babs) [reply src/collections/blog/2021-04-16-meet-the-maintainer-jash-patel/index.mdx:125]: Okay, I'll do that

**Comment by blossom-babs** [comment]
> @blossom-babs any updates here? If you are having trouble resolving the merge conflict, you can open up a new PR

Yes I think I'll do that now, thanks!

**Comment by Nikhil-Ladha** [comment]
@blossom-babs any updates for the new PR?

**Comment by l5io** [comment]
🚀 Preview for commit b1b2878d6eb052baacba7df4e332ed2a891cbebb at: https://60903398e6a15300bef7ed04--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
No worries, @blossom-babs I did it for you :)

**Comment by l5io** [comment]
🚀 Preview for commit d56f251dff02198d9502db63d14426a20b1d47d9 at: https://6090360222397212d2514c14--layer5ng.netlify.app
